Source/Seal.spoon/seal_filesearch.lua

```diff
@@ -20,54 +20,163 @@ obj.maxQueryResults = 40
 
 --- Seal.plugins.filesearch.displayResultsTimeout
 --- Variable
---- Maximum time to wait before displaying the results
---- Defaults to 0.2s (200ms).
+--- Maximum time to wait before displaying the results. Defaults to 0.2s (200ms).
 ---
 --- Notes:
 ---  * higher value might give you more results but will give a less snappy experience
 obj.displayResultsTimeout = 0.2
 
--- Variables
-obj.currentQuery = nil
-obj.currentQueryResults = {}
-obj.currentQueryResultsDisplayed = false
-obj.showQueryResultsTimer = nil
+--- Seal.plugins.filesearch.queryChangedTimerDuration
+--- Variable
+--- Maximum delay after typing a query before initiating a file search. Defaults to 0.1s (100ms).
+---
+--- Notes: This plugin uses its own parameter, separate from the Seal one, as each query
+---        initiates a new spotlight search that is more resource-intensive than other plugins.
+---        To prevent impacting interface responsiveness, a value higher than the default Seal
+---        one is typically desired.
+obj.queryChangedTimerDuration = 0.1
 
-obj.spotlight = hs.spotlight.new()
+---
```

Comment:
Having `---` here makes the docstrings parser think you're starting a new entry, but there is nothing specified. This line and line 41 need to be reduced to two dashes.